### ISSUE TITLE
File Integrity Checksum changed reporting issue

### ISSUE THREAD

**Issue by NovaDear** [ossec/ossec-hids#1581]
We have Ossec installed across our environment and recently updated to Server and Agent 3.0. Since the update our windows agents have been reporting changes on a lot of files that when we look into it look like below;

2018 Nov 23 10:43:18 (SERVERNAME)->syscheck
Rule: 550 (level 7) -> 'Integrity checksum changed.'
Integrity checksum changed for: 'C:\Windows/SysWoW64/0409/scregedit.ini'
Size changed from 'GetSecurityInfo=8' to 'SID String'

We are also seeing CreateFile=32 in place of GetSecurityInfo=8 as well.

Clearly this isn't the size of the file that is being reported. We have tried downgrading the agent back to 2.9.4 and it continues to report these. Anyone else experienced this?

**Comment by NovaDear** [comment]
Correct, our Linux hosts look good but all of our windows hosts are reporting this.

**Comment by ddpbsd** [comment]
I managed to scrounge up a Windows agent. I haven't triggered any syscheck alerts yet, but the syscheck db looks fine. 
Is the alerts.log affected, or is it just emailed alerts?

**Comment by NovaDear** [comment]
Hello,

It is the alerts.log affected as well. It's counting this string as a size change for a file and triggering a Size Changed alert in alerts.log

**Comment by NovaDear** [comment]
Just wondering if there was any progress on this bug?
